cou/logging.py

```diff
@@ -39,15 +40,25 @@ def filter(self, record: logging.LogRecord) -> bool:
         return True
 
 
-def setup_logging(log_level: str = "INFO") -> None:
+def get_log_file() -> Path:
+    """Get log file path.
+
+    :return: Returns log file path
+    :rtype: Path
+    """
+    time_stamp = datetime.now().strftime("%Y%m%d%H%M%S")
+    return Path(f"{COU_DIR_LOG}/cou-{time_stamp}.log")
+
+
+def setup_logging(log_file: Path, log_level: str = "INFO") -> None:
     """Do setup for logging.
 
+    :param log_file: Logging file, defaults to "{COU_DIR_LOG}/cou-{time_stamp}.log"
+    :type log_level: Path
     :param log_level: Logging level, defaults to "INFO"
     :type log_level: str, optional
     """
     progress_indicator.start("Configuring logging...")
```

Comment:
In this line we start the indicator with message: "Configuring logging ..." and you remove part, when we stop it. Can you put it back?

Reply:
Added back, so resolved.